### ISSUE TITLE
Consider adding WebP and AVIF versions

### ISSUE THREAD

**Issue by mathiasbynens** [microsoft/fluentui-emoji#31]
The repository hosts PNG versions of the assets. Have you considered providing WebP and AVIF versions as well?

**Comment by filiptronicek** [comment]
I like the idea, @mathiasbynens, but since those are both image formats for the web, wouldn't it still be better to just use SVGs there?

**Comment by mathiasbynens** [comment]
> I like the idea, @mathiasbynens, but since those are both image formats for the web, wouldn't it still be better to just use SVGs there?

Of course. I don’t see SVG versions of the 3D flavors in this repository either, though.

**Comment by yisibl** [comment]
Have you considered adding a version with animations?

**Comment by justsomederpystuff** [comment]
> > I like the idea, @mathiasbynens, but since those are both image formats for the web, wouldn't it still be better to just use SVGs there?
> 
> Of course. I don’t see SVG versions of the 3D flavors in this repository either, though.

"3D" svg variants are in the /Color/ directory
I'd still like to see more image formats since the svgs especially due to their complexity tend to cause slowdown when used in higher quantities.
